Formula/openiked.rb

```diff
@@ -0,0 +1,54 @@
+class Openiked < Formula
+  desc "IKEv2 daemon - portable version of OpenBSD iked"
+  homepage "https://openiked.org"
+  url "https://cdn.openbsd.org/pub/OpenBSD/OpenIKED/openiked-7.2.tar.gz"
+  mirror "https://mirror.edgecast.com/pub/OpenBSD/OpenIKED/openiked-7.2.tar.gz"
+  sha256 "55dc270bc40a121f855d949a25a5ffaeb11e7607e8198ec52160ef54b6946845"
+  license "ISC"
+
+  depends_on "cmake" => :build
+  depends_on "libevent"
+  depends_on "openssl@1.1"  # openssl@3 conflicts with libevent
+
+  on_linux do
+    depends_on "bison"
+  end
+
+  def install
+    system "cmake", "-DHOMEBREW=true",
+                    "-DCMAKE_BUILD_TYPE=Release",
+                    "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}",
+                    "-DCMAKE_INSTALL_MANDIR=#{man}"
+    system "make"
+    bin.install "iked/iked", "ikectl/ikectl"
+    libexec.install "regress/dh/dhtest"
+    libexec.install "setup_config.sh"
+    etc.install "iked.conf"
+  end
+
+  def post_install
+    system "#{libexec}/setup_config.sh", "#{etc}/iked"
```

Comment:
```suggestion
    system libexec/"setup_config.sh", "#{etc}/iked"
```

Reply:
ok

Reply:
fixed: not needed when using `make install`